easybuild/easyconfigs/x/XMDS2/XMDS2-2.2.3-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'Bundle'
```

Comment:
@hajgato Do you mind changing this to the new `PythonBundle`, which will be included in EasyBuild v3.8.0 (cfr. https://github.com/easybuilders/easybuild-easyblocks/pull/1553)?